### DISCUSSION TITLE
How to handle Server Sent Event

### DISCUSSION THREAD

**Discussion by Ikki-Dai** [axios/axios#5881] [Q&A]
### Describe the issue

what if to process **Server Sent Event**, what does field `reponseType` set?

seems only `onDownloadProgress ` can process

**Comment by ishowshao** [comment]
great issue

**Comment by LassazVegaz** [comment]
No updates?

**Comment by darrylhebbes** [comment]
Axios is not optimised to handle "Long lived events". It could but its a stretch.

That being said you could hack to make  Axios handle the SSE stream (stream headers), parse stream events, manage readyState and handle errors (retries). 

Or find a library (https://github.com/EventSource/eventsource) that specialises in doing this or simply implement a basic wrapper around [EventSource](https://developer.mozilla.org/en-US/docs/Web/API/EventSource )
